src/main/java/org/fcrepo/sword/http/SWORDServiceResource.java

```diff
@@ -28,15 +31,25 @@
  */
 @Scope("request")
 @Path("/sword")
```

Comment:
No, the OAI provider needs to be fixed as well... the OAI provider issue is what made me also notice it here.